docs/root/install/getting_started.rst

```diff
@@ -0,0 +1,30 @@
+.. _install_getting_started:
+
+Getting Started
```

Comment:
What about including the google.com smoketest in this section as well? https://www.envoyproxy.io/docs/envoy/latest/install/ref_configs.html#smoketest-configuration

Reply:
+1. We should move that to here from that buried location and highlight it. The smoke test config is very simple to understand and a great way to get started.

Reply:
No problem. Not sure if you guys saw what I wrote at https://www.envoyproxy.io/download when I did the website, but I can copy the docs I wrote into here. (At least, when I came across the smoketest, I had to spend awhile figuring out how to get the Envoy binary to actually run the config.)